circle-mlir/circle-mlir/lib/pass/src/ConvertONNXToCirclePass.cpp

```diff
@@ -165,6 +166,8 @@ void ConvertONNXToCirclePass::runOnOperation()
   patterns.insert<ConvNone>(typeConverter, context);
   patterns.insert<ConvPad>(typeConverter, context);
   patterns.insert<ConvPRelu>(typeConverter, context);
+  patterns.insert<ConvReduceMean>(typeConverter, context);
+  patterns.insert<ConvReduceMeanV13>(typeConverter, context);
```

Comment:
`ConvReduceMean` is for latest version and `ConvReduceMeanV13` for specific version, 13,
to support old one, that is, latest version > specific version